src/lib/ol3cesium-debug.js

```diff
@@ -27648,21 +27648,37 @@ olcs.FeatureConverter.prototype.olLineStringGeometryToCesium = function(layer, f
     material: this.olStyleToCesium(feature, olStyle, true)
   });
 
-  // Handle both color and width
-  var outlineGeometry = new Cesium.PolylineGeometry({
+  var geometryOptions = {
     // always update Cesium externs before adding a property
     positions: positions,
     width: this.extractLineWidthFromOlStyle(olStyle),
     vertexFormat: appearance.vertexFormat
-  });
+  };
+
+  var outlinePrimitive;
+  var heightReference = this.getHeightReference(layer, feature, olGeometry);
+
+  if (heightReference == Cesium.HeightReference.CLAMP_TO_GROUND) {
+    var color = this.extractColorFromOlStyle(olStyle, true);
+    outlinePrimitive = new Cesium.GroundPrimitive({
+      // always update Cesium externs before adding a property
+      geometryInstances: new Cesium.GeometryInstance({
+        geometry: new Cesium.CorridorGeometry(geometryOptions),
```

Comment:
what about provinding a height to `Cesium.CorridorGeometry`.
I quote:
The distance in meters between the ellipsoid surface and the positions.

So the altitude or height in out case, defaults to 0.